### PR TITLE
Create wptuning.json

### PR DESCRIPTION
**Pull request by lriaudel**
## Please select a type for your pull request (PR)

[] My PR is about registering`*` a plugin to the Entrepôt repositories. Doing it, I'm agreeing with the [code of conduct](https://github.com/imath/entrepot/blob/master/CODE_OF_CONDUCT.md).
[] My PR is regular and is about the plugin's code improvement.

`*` Read the Plugin author's [documentation](https://github.com/imath/entrepot/wiki) for more information.

### CODE REVIEW

**Comment by imath** [comment]
Hi!

I think you need to use `wp-tuning` into the slug property of your json file to have tests passed. Could you update your PR ? I’ll be very happy to merge it once done.

**Comment by imath** [comment]
Weird the test is still failing. I'll give it a closer look asap (sorry for this extra delay).

File: repositories/wptuning.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+		"cleaning"
+	],
+	"country": "en_US",
+	"releases": "https://github.com/lriaudel/wp-tuning/releases",
+	"issues": "https://github.com/lriaudel/wp-tuning/issues",
+	"description": {
+		"en_US" : "Differents hooks and functions to improve WordPress and basic customization in a plugin.",
+		"fr_FR" : "Differents hooks et fonctions pour améliorer WordPress dans un plugin !"
+	},
+	"README": "https://github.com/lriaudel/wp-tuning/blob/master/README.md"
```
Comment (imath) [inline repositories/wptuning.json]: The README property should be: `https://raw.githubusercontent.com/lriaudel/wp-tuning/master/README.md`

File: repositories/wptuning.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+	"name": "WP Tuning",
+	"author": "lriaudel",
+	"slug": "wp-tuning",
+	"icon": "https://raw.githubusercontent.com/lriaudel/wp-tuning/master/wptuning.svg",
```
Comment (imath) [inline repositories/wptuning.json]: The icon should be a png file for better results.

File: repositories/wptuning.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (imath) [inline repositories/wptuning.json]: The file name should be wp-tuning.json to be consistent with the slug property.

**Review by imath — CHANGES_REQUESTED** [review]
Hi i've tested on my local dev. Unit tests are failing because the filename is not built using the slug property. It should be `{slug}.json` eg: `wp-tuning.json`.

I've also added 2 advices about the readme and the icon properties for better display results in Entrerpôt Administration screens.

File: repositories/wptuning.json (hunk 3 above)
Comment (madvik) [reply repositories/wptuning.json]: OK, :-(
I change the slug because it exist an another plugin : https://wordpress.org/plugins/wp-tuning/

Comment (lriaudel) [reply repositories/wptuning.json]: I change the slug because it exist an another plugin with same slug : https://wordpress.org/plugins/wp-tuning/

**Comment by lriaudel** [comment]
Et bien ! Ca n'a pas été facile :-b !

**Review by imath — APPROVED** [review]

**Pull request merged**